code/__DEFINES/subsystems.dm

```diff
@@ -48,7 +48,7 @@
 // Subsystems shutdown in the reverse of the order they initialize in
 // The numbers just define the ordering, they are meaningless otherwise.
 
-#define INIT_ORDER_GARBAGE 19
+
```

Comment:
code/controllers/subsystem/garbage.dm:7:error: INIT_ORDER_GARBAGE: undefined var

You removed this define